core/object/object.h

```diff
@@ -61,6 +61,7 @@ enum PropertyHint {
 	PROPERTY_HINT_LAYERS_3D_RENDER,
 	PROPERTY_HINT_LAYERS_3D_PHYSICS,
 	PROPERTY_HINT_LAYERS_3D_NAVIGATION,
+	PROPERTY_HINT_LAYERS_AVOIDANCE,
```

Comment:
Adding a new member in the middle of the enum breaks compatibility, new members should be added at the end.